components/FlexibleForms/RepeaterGroupField.test.tsx

```diff
@@ -0,0 +1,144 @@
+import RepeaterGroupField from './RepeaterGroupField';
+import { Formik } from 'formik';
+import { render, screen } from '@testing-library/react';
+
+const mockSubmit = jest.fn();
+
+describe('RepeaterGroupField', () => {
```

Comment:
We could do with a test/s for adding.removing fields.

Reply:
done